### PR TITLE
JR2-1194 Return system notification id in create response

### PR DESCRIPTION
**Pull request by varusakov**

### CODE REVIEW

File: src/main/java/ch/admin/seco/jobroom/service/SystemNotificationService.java [hunk 1]
```diff
@@ -16,70 +16,56 @@
...
 
 @Service
 @Transactional
 public class SystemNotificationService {
 
-    private SystemNotificationRepository systemNotificationRepository;
     private final Logger log = LoggerFactory.getLogger(SystemNotificationService.class);
 
-    public SystemNotificationService(SystemNotificationRepository systemNotificationRepository) {
+    private SystemNotificationRepository systemNotificationRepository;
```
Comment (birom) [inline src/main/java/ch/admin/seco/jobroom/service/SystemNotificationService.java]: Why do we need all these changes? For me it seems that we are having some issues with the time zone handling.
Does the new mapper solve this?

Comment (varusakov) [reply src/main/java/ch/admin/seco/jobroom/service/SystemNotificationService.java]: @birom Issue with time handling was on front end

Comment (birom) [reply src/main/java/ch/admin/seco/jobroom/service/SystemNotificationService.java]: @varusakov If we do not need the other changes please revert them. Thank you!

File: src/main/webapp/app/shared/utils/date-utils.ts [hunk 2]
```diff
@@ -49,4 +49,10 @@ export class DateUtils {
     static convertNgbDateStructToString(date: NgbDateStruct): string {
         return date ? moment(DateUtils.mapNgbDateStructToDate(date)).format('YYYY-MM-DD') : null;
     }
+
+    static convertNgbDateTimeToISOWithoutZone(date: NgbDateStruct, time: NgbTimeStruct): string {
+        return moment().utc().date(date.day).month(date.month - 1).year(date.year)
```
Comment (birom) [inline src/main/webapp/app/shared/utils/date-utils.ts]: Does it make any difference if we use ```moment().utc()``` or just simply ```moment()``` when we do the formatting with the ```YYYY-MM-DDTHH:mm:ss.sss``` pattern?

File: src/main/java/ch/admin/seco/jobroom/service/SystemNotificationService.java (hunk 1 above)
Comment (varusakov) [reply src/main/java/ch/admin/seco/jobroom/service/SystemNotificationService.java]: @birom we already have concept of mappers in jobroom. So I think it's good idea to move object mapping from service to them.

File: src/main/webapp/app/shared/utils/date-utils.ts (hunk 2 above)
Comment (varusakov) [reply src/main/webapp/app/shared/utils/date-utils.ts]: Thank you, you are right. Simple `moment()` works perfectly

File: src/main/java/ch/admin/seco/jobroom/service/SystemNotificationService.java (hunk 1 above)
Comment (birom) [reply src/main/java/ch/admin/seco/jobroom/service/SystemNotificationService.java]: @cjpsycos What do you think?

Comment (cjpsycos) [reply src/main/java/ch/admin/seco/jobroom/service/SystemNotificationService.java]: I hate this mapper things. It is less readable and usually change-errors are seen, when it's deployed. with manually mapping you see what is going where and e.g. type-changes are recognized instantly.

Comment (varusakov) [reply src/main/java/ch/admin/seco/jobroom/service/SystemNotificationService.java]: Ok, I'll revert this changes

**Pull request merged**